packages/os/early-boot-config.service

```diff
@@ -2,10 +2,15 @@
 Description=Bottlerocket userdata configuration system
 # Need network online to talk to IMDS.
 After=network-online.target apiserver.service
-Requires=network-online.target apiserver.service
+Requires=network-online.target
+# We don't want to restart the unit if apiserver restarts
+Wants=apiserver.service
```

Comment:
`network-online.target` shouldn't be fallible but the same rationale would apply to it:
```suggestion
# We don't want to restart the unit if the network goes offline or apiserver restarts
Wants=network-online.target apiserver.service
```